src/main/scala/rocket/RocketCore.scala

```diff
@@ -241,6 +248,7 @@ class Rocket(tile: RocketTile)(implicit p: Parameters) extends CoreModule()(p)
   val ex_reg_inst = Reg(Bits())
   val ex_reg_raw_inst = Reg(UInt())
   val ex_reg_wphit            = Reg(Vec(nBreakpoints, Bool()))
+  val ex_reg_set_vconfig      = Reg(Bool())
```

Comment:
I wanted to avoid pipeline bubbles due to dependency on new VL.

Reply:
The other key requirement is for performant precise traps. The checks for memory access fault in M/W stages must occur ahead of the commit point and depend on the updated vconfig.

Reply:
> I wanted to avoid pipeline bubbles due to dependency on new VL.

Understand, we do it in another way: for each vsetvl, it will be a phantom instruction that will be observed by following instructions. But we didn't take the precise trap into consideration.